source/distributed/shared_tria.cc

```diff
@@ -134,6 +134,16 @@ namespace parallel
       this->update_number_cache ();
     }
 
+    template <int dim, int spacedim>
+    void
+    Triangulation<dim, spacedim>::
+    copy_triangulation (const dealii::Triangulation<dim, spacedim> &old_tria)
```

Comment:
this probably only works when input Tria is **not** parallel::distributed::Tria. I would add an assert